### PR TITLE
Template sets findbugs and chromium-cronet

### PR DESCRIPTION
**Pull request by moljac**
### Describe your contribution

1. Moved `Xamarin.Chromium.CroNet.*` to this repo from:
   https://github.com/xamarin/XamarinComponents/tree/main/XPlat/Chromium.CroNet
2. created template set

### Does this change any of the generated binding API's?

No.

### Artifacts updated

- org.chromium.net:cronet-api - 76.3809.111 -> 72.3626.96

  - [diff.md.txt](https://github.com/xamarin/GooglePlayServicesComponents/files/7343586/Xamarin.Chromium.CroNet.Api.dll.diff.md.txt)

- org.chromium.net:cronet-common -  -> 72.3626.96

  - [diff.md.txt](https://github.com/xamarin/GooglePlayServicesComponents/files/7343601/Xamarin.Chromium.CroNet.Common.dll.diff.md.txt)

- org.chromium.net:cronet-embedded -  -> 72.3626.96

  - [diff.md.txt](https://github.com/xamarin/GooglePlayServicesComponents/files/7343603/Xamarin.Chromium.CroNet.Embedded.dll.diff.md.txt)

  - [breaking.md.txt](https://github.com/xamarin/GooglePlayServicesComponents/files/7343602/Xamarin.Chromium.CroNet.Embedded.dll.breaking.md.txt)

- org.chromium.net:cronet-fallback -  -> 72.3626.96

  - [diff.md.txt](https://github.com/xamarin/GooglePlayServicesComponents/files/7343606/Xamarin.Chromium.CroNet.Fallback.dll.diff.md.txt)

  - [breaking.md.txt](https://github.com/xamarin/GooglePlayServicesComponents/files/7343605/Xamarin.Chromium.CroNet.Fallback.dll.breaking.md.txt)

### Artifacts new

- com.google.code.findbugs:jsr305 -  -> 3.0.1

  - [diff.md.txt](https://github.com/xamarin/GooglePlayServicesComponents/files/7343594/Xamarin.Google.Code.FindBugs.JSR305.dll.diff.md.txt)

  - [breaking.md.txt](https://github.com/xamarin/GooglePlayServicesComponents/files/7343593/Xamarin.Google.Code.FindBugs.JSR305.dll.breaking.md.txt)

Checklist
- [x] Add any metadata needed to get PR to compile
- [x] Verify any new namespaces are properly renamed (utilities.cake)
- [x] Verify nuget/namespace spelling (utilities.cake)           
- [x] Locally generate diffs
- [x] Review any breaking changes
- [x] Attach diffs to PR

### CODE REVIEW

**Comment by moljac** [comment]
binderator needs fixing, because it does not support interpolation and variables

https://maven.apache.org/guides/introduction/introduction-to-the-pom.html#project-interpolation-and-variables

error with added `org.chromium.net:cronet*` artifacts to `config.json`:

```
Dependency errors : 3
1
	System.Exception: 
No matching artifact config found for: 
			${project.groupId}.cronet-api:${project.version}
to satisfy dependency of: 
			org.chromium.net.cronet-common:72.3626.96

	Please add following json snippet to config.json:


      {
        "groupId": "${project.groupId}",
        "artifactId": "cronet-api",
        "version": "${project.version}",
        "nugetVersion": "CHECK NUGET ID",
        "nugetId": "CHECK PREFIX ${project.version}",
        "dependencyOnly": true/false
      }
						


2
	System.Exception: 
No matching artifact config found for: 
			${project.groupId}.cronet-common:${project.version}
to satisfy dependency of: 
			org.chromium.net.cronet-embedded:72.3626.96

	Please add following json snippet to config.json:


      {
        "groupId": "${project.groupId}",
        "artifactId": "cronet-common",
        "version": "${project.version}",
        "nugetVersion": "CHECK NUGET ID",
        "nugetId": "CHECK PREFIX ${project.version}",
        "dependencyOnly": true/false
      }
						


3
	System.Exception: 
No matching artifact config found for: 
			${project.groupId}.cronet-common:${project.version}
to satisfy dependency of: 
			org.chromium.net.cronet-fallback:72.3626.96

	Please add following json snippet to config.json:


      {
        "groupId": "${project.groupId}",
        "artifactId": "cronet-common",
        "version": "${project.version}",
        "nugetVersion": "CHECK NUGET ID",
        "nugetId": "CHECK PREFIX ${project.version}",
        "dependencyOnly": true/false
      }

```

because of

```
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">

  <modelVersion>4.0.0</modelVersion>

  <groupId>org.chromium.net</groupId>

  <artifactId>cronet-common</artifactId>

  <version>72.3626.96</version>

  <packaging>aar</packaging>

  <name>Cronet</name>

  <description>Cronet shared implementation. Not a complete implementation; apps and libraries should not depend on this target directly.</description>

  <dependencies>

    <dependency>

      <groupId>${project.groupId}</groupId>

      <artifactId>cronet-api</artifactId>

      <version>${project.version}</version>

    </dependency>

  </dependencies>

  <licenses>

    <license>

      <name>Chromium and built-in dependencies</name>

      <url>https://storage.cloud.google.com/chromium-cronet/android/72.0.3626.96/Release/cronet/LICENSE</url>

      <distribution>repo</distribution>

    </license>

  </licenses>

</project>
```

File: config.json [hunk 1]
```diff
@@ -1419,15 +1419,52 @@
...
+        "nugetId": "Xamarin.Google.Code.FindBugs.JSR305",
+        "dependencyOnly": false,
+        "templateSet": "findbugs"      
+      },
       {
         "groupId": "org.chromium.net",
         "artifactId": "cronet-api",
-        "version": "76.3809.111",
-        "nugetVersion": "76.3809.111",
+        "version": "72.3626.96",
```
Comment (jpobst) [inline config.json:1434]: The most recent NuGets for all of the Cronet's is `76.3809.111`.  Is there a reason bind an older version instead?

File: templates/chromium-cronet/Project.cshtml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Authors>Microsoft</Authors>
+    <Owners>Microsoft</Owners>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=865435</PackageLicenseUrl>
+    <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=865435</PackageProjectUrl>
+    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=865373</PackageLicenseUrl>
+    <RepositoryUrl>https://github.com/tensorflow/tensorflow/</RepositoryUrl>
```
Comment (jpobst) [inline templates/chromium-cronet/Project.cshtml:29]: I think this needs to be updated.

File: templates/findbugs/Project.cshtml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Authors>Microsoft</Authors>
+    <Owners>Microsoft</Owners>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=865435</PackageLicenseUrl>
+    <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=865435</PackageProjectUrl>
+    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=865373</PackageLicenseUrl>
+    <RepositoryUrl>https://github.com/tensorflow/tensorflow/</RepositoryUrl>
```
Comment (jpobst) [inline templates/findbugs/Project.cshtml:29]: Same

File: config.json [hunk 4]
```diff
@@ -1419,15 +1419,52 @@
         "nugetId": "Xamarin.Google.Dagger",
         "dependencyOnly": true
       },
+      {
+        "groupId": "com.google.code.findbugs",
+        "artifactId": "jsr305",
+        "version": "3.0.1",
+        "nugetVersion": "3.0.1",
```
Comment (jpobst) [inline config.json:1426]: The most recent version on NuGet is `3.0.2`.

**Review by jpobst — COMMENTED** [review]
The filename for FindBugs is different than the one on NuGet, not sure if this matters:

image

File: templates/chromium-cronet/Project.cshtml (hunk 2 above)
Comment (moljac) [reply templates/chromium-cronet/Project.cshtml:29]: tnesorflow only. This is from 1st template set PR.

Other urls point to GPS-FB-MLKit repo.

fixed:
http://findbugs.sourceforge.net/

File: config.json (hunk 4 above)
Comment (moljac) [reply config.json:1426]: completely new artifact in `config.json` so I messed something up.

fixed

**Review by jpobst — APPROVED** [review]

**Pull request merged**